client/src/main/java/io/pravega/client/admin/StreamInfo.java

```diff
@@ -45,4 +46,9 @@
      * {@link StreamCut} representing the current HEAD of the stream.
      */
     private final StreamCut headStreamCut;
+
+    /**
+     * Represents the stream state. It is set to True if the stream is Sealed.
```

Comment:
I wouldn't say "stream state". Maybe at this point sealed is the only way to describe a Stream's state, but maybe in the future we want to add some other info. I would keep this simple. For example: "Indicates whether the Stream is sealed (true) or not (false). If a stream is sealed, the no further Events can be written to it." ( <-- feel free to include any other operations that will be denied)

Reply:
thanks, improved the comment.